iron-ajax.html

```diff
@@ -172,6 +172,8 @@
        *    `blob`: uses `XHR.response`.
        *
        *    `document`: uses `XHR.response`.
+       *
+       *    `auto`: select data automatically.
```

Comment:
Recommend instead of "select data automatically" something more like "one of the above, inferred by the received Content-Type"